controls/postgres_spec.rb

```diff
@@ -131,10 +64,15 @@
   impact 1.0
   title 'Run one postgresql instance per operating system'
   desc 'Only one postgresql database instance must be running on an operating system instance (both physical HW or virtualized).'
-  pg_command = 'postgres'
-  pg_command = 'postmaster' if os.redhat? && os.release.include?('6.')
-  describe processes(pg_command) do
```

Comment:
I think, we should keep using the `process` ressource (https://docs.chef.io/inspec/resources/processes/). There is a feature to check for number of found processes.

Reply:
Thanks @schurzi, will be changed.